util/sdks/go.sh

```diff
@@ -128,4 +133,5 @@ if [ "$mode" = "download" ]; then
 
 else
     build_go_sdk "$version" "$clean"
+    install_go_sdk_examples
```

Comment:
Shouldn't `install_go_sdk_examples` also be invoked in the "download" mode?

Reply:
Ditto: download mode

Reply:
No, because AFAICT that's for downloading the SDK only. The idea was only to factor out the install step out of build so that one could skip rebuild without skipping install (hence return instead of exit)

Reply:
Hmm right ok I looked at the full script and I see now, it isn't exactly like `runtime.sh`. Ok then! I'll merge with the `return` change.

Reply:
Also just to leave a note on it: I'm not sure we need this "download" mode for SDKs... It's harmless for now but the more SDKs we add, the more likely we'll prune it out because it'll keep the scripts lighter imho.